src/api/admin.ts

```diff
@@ -129,6 +129,20 @@ export interface VenueInput_v2
   end_utc_seconds?: number;
 }
 
+export interface WorldProps {
+  id: string;
+  name: string;
+  description?: string;
+  subtitle?: string;
+  bannerImageFile?: FileList;
+  bannerImageUrl?: string;
+  logoImageFile?: FileList;
+  logoImageUrl?: string;
```

Comment:
there types are a bit mismatching based on what I've seen in `functions/world`. Why?

Reply:
Because these are the form values that are expected in the create function, I will create another interface for the actual object thats being returned from the database.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2295/commits/563960b617e371ddfc9d465b3428bdae64fafb0d